kanidmd/src/lib/credential/mod.rs

```diff
@@ -218,6 +219,17 @@ pub struct BackupCodes {
     code_set: HashSet<String>,
 }
 
+impl fmt::Display for BackupCodes {
```

Comment:
Where is this display used? Do we need the server to be able to display/log these? I think that could be a security issue if we log someones backup codes ....

Reply:
Not used, this was a wrong idea. I will remove it.